components/Login/SignIn.tsx

```diff
@@ -32,26 +36,57 @@ interface FormValues {
   checked: Array<string>
 }
 
+// checks if the user is able to log in
+export function login(data: Session | null, router: NextRouter) {
+  if (data?.user) {
+    if (data.user.id) {
+      router.push('/DynamicPOC')
+    } else {
+      alert(data.user.name)
```

Comment:
is there any reason for this to happen rn? Like if the user doesn't have an id we don't need to alert, we just need to error right?